modules/ebi-metagenomics/proovframe/map/meta.yml

```diff
@@ -0,0 +1,57 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "proovframe_map"
+description: "frame-shift correction for long read (meta)genomics"
+keywords:
+  - "frame-shift correction"
+  - "long-read sequencing"
+  - "sequence analysis"
+tools:
+  - "proovframe":
+      description: "frame-shift correction for long read (meta)genomics"
+      homepage: "https://github.com/thackl/proovframe"
+      documentation: "https://github.com/thackl/proovframe"
+      tool_dev_url: "https://github.com/thackl/proovframe"
+      doi: "10.1101/2021.08.23.457338"
+      licence: ["MIT"]
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - faa:
+      type: file
+      description: "A FASTA file containing protein sequences"
```

Comment:
let's update to: "A FASTA file containing a database of guide protein sequences"